src/tests/unit/auto/auto_select_device_failed_test.cpp

```diff
@@ -275,28 +275,20 @@ TEST_P(AutoLoadFailedTest, LoadCNNetWork) {
 // so inference request num is 4 (CPU 2, MYRIAD 2)
 //
 const std::vector<ConfigParams> testConfigs = {ConfigParams {true, false, GENERAL, {DeviceParams {CommonTestUtils::DEVICE_GPU, true},
-                                                        DeviceParams {"MYRIAD", true},
```

Comment:
ticket 101367 is created to follow up any Myriad/HDDL related change in AUTO/MULTI